packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -62,6 +65,18 @@ export interface ParticipantItemStrings {
   mutedIconLabel: string;
   /** placeholder text for participants who does not have a display name*/
   displayNamePlaceholder?: string;
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** String shown when `participantState` is `Connecting` */
+  participantStateConnecting?: string;
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** String shown when `participantState` is `Ringing` */
+  participantStateRinging?: string;
```

Comment:
ideally this component is agnostic of being chat or calling...
Thinking out loud so feel free to shut this down - Instead of passing participantState into here, would it be possible to pass any string that is simply written to the right hand side of the participant item, `participantItemSecondaryText?: string` - then the VideoGalleryParticipant does the mapping of state->string and passes in the appropriate string

Reply:
I understand your point of view. And it's a good idea.
However, that removes the localization support we offer for these connection states.
There's another concern of customer injected strings being too long and messing up the UI. They would then also have to deal with truncation.
By adding a fixed set of connection states, we keep this component communication specific.

Reply:
To keep the locale strings, would the localization not live in the videoGallery component locale?

Reply:
I think we need to handle truncation regardless and show an ellipsis or something for long text as we don't know what our translated strings will, or what width Contoso will set the participantItem to be

Reply:
Good point.

Reply:
But does that mean that now we have to introduce these strings to VideoGallery props as well?

Reply:
They would be introduced there *instead (they should only need to be in one place, but I haven't dug into the code)

Reply:
We chatted offline and making this non-blocking for cost to implement this change.. but I still strongly feel this should happen, for posterity in case someone does decide to make this change reiterating why:
* Not pollute participantItem with calling-only properties
* Keep the property as simple as possible to ensure ease of use and understanding by Contoso devs (nothing surprising can happen like hiding the menu button)
* Keeps it flexible - e.g. if Contoso wanted "sharing screen" text here or something (which we have an icon for but just an example)